src/server/pegasus_server_impl.cpp

```diff
@@ -438,24 +438,24 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
         if (c > 0 || (c == 0 && (!start_inclusive || !stop_inclusive))) {
             // empty sort key range
             if (_verbose_log) {
-                LOG_WARNING(
-                    "%s: empty sort key range for multi_get from %s: hash_key = \"%s\", "
-                    "start_sort_key = \"%s\" (%s), stop_sort_key = \"%s\" (%s), "
-                    "sort_key_filter_type = %s, sort_key_filter_pattern = \"%s\", "
-                    "final_start = \"%s\" (%s), final_stop = \"%s\" (%s)",
+                LOG_WARNING_F(
```

Comment:
Use `LOG_WARNING_PREFIX` instead:
```suggestion
                LOG_WARNING_PREFIX(
```